cue/cue.mod/p2plab.cue

```diff
@@ -0,0 +1,40 @@
+// defines a set of nodes of size 1 or higher
+// a "node" is simply an EC2 instance provisioned of the given type
+Nodes :: {
```

Comment:
I think this should just be called `Group`.

Reply:
Good idea, I did find `Nodes` a bit awkward.